docs/operations/cluster-id.md

```diff
@@ -1,26 +1,58 @@
 ---
-title: "Obtain a ClusterID"
-linkTitle: Obtain a ClusterID
+title: "How To Get Your Ondat Cluster ID"
+linkTitle: "How To Get Your Ondat Cluster ID"
 ---
 
-Every Ondat cluster has a unique ClusterID generated at bootstrap. An Ondat Licence is specific for a ClusterID.
+## Overview
 
-## How to obtain the ClusterID
+Every Ondat cluster has a unique cluster ID that is generated once an Ondat deployment is successfuly completed. In order for end users to be able to generate an Ondat licence and apply it to the cluster, an Ondat cluster ID is required.
 
-You can obtain the ClusterID using the CLI.
+## Prerequisites
 
-This CLI command can print the cluster ID:
+- Ensure that you have successfully [installed Ondat](/docs/install/) into your Kubernetes or Openshift cluster.
 
-```bash
-$ storageos get cluster
-ID:               704dd165-9580-4da4-a554-0acb96d328cb
-Licence:
-  expiration:     2021-03-25T13:48:46Z (1 year from now)
-  capacity:       5.0 TiB
-  kind:           professional
-  customer name:  storageos
-Created at:       2020-03-25T13:48:33Z (1 hour ago)
-Updated at:       2020-03-25T13:48:46Z (1 hour ago)
-```
+## Procedure
 
-Given the Cluster ID, you can generate a license on the [SaaS Platform](https://portal.ondat.io/) following [this procedure](/docs/operations/licensing/#procedure).
+### Step 1 - Install The Ondat CLI
+
+- Deploy and run the [Ondat CLI utility as a deployment](https://docs.ondat.io/docs/reference/cli/#run-the-cli-as-a-deployment-in-your-cluster) first, so that you can interact and manage Ondat alongside  `kubectl`. Once deployed, obtain the Ondat CLI utility pod name for later reference.
+
+  ```bash
+  # Get the Ondat CLI utility pod name.
+  kubectl --namespace=storageos get pod -ocustom-columns=_:.metadata.name --no-headers -lapp=storageos-cli
+  
+  storageos-cli-6958b7c4cd-nc8q5
+  ```
+
+### Step 2 - Locate The Ondat Cluster ID
+
+- With the Ondat CLI deployed, you can run one of the following commands below to get the unique ID of the cluster.
+  - The cluster ID will printed out as a UUID value with either a `ID` or `ClusterID` key.
+
+  ```bash
+  # Get cluster-wide information
+  kubectl --namespace=storageos exec storageos-cli-6958b7c4cd-nc8q5 -- storageos get cluster
+
+  ID:           ce7c9c90-896e-41d5-b823-27ed9a8d8c8f
+  Created at:   2022-08-23T16:24:33Z (10 minutes ago)
+  Updated at:   2022-08-23T16:24:33Z (10 minutes ago)
+  Nodes:        5
+    Healthy:    5
+    Unhealthy:  0
+
+  # Get more information about the licence status and unique cluster ID.
+  kubectl --namespace=storageos exec storageos-cli-6958b7c4cd-nc8q5 -- storageos get licence
+
+  ClusterID:      ce7c9c90-896e-41d5-b823-27ed9a8d8c8f
+  Expiration:     2022-08-24T16:24:33Z (23 hours from now)
+  Capacity:       50 GiB (53687091200)
+  Used:           0 B (0)
+  Kind:           unregistered
+  Features:       []
+  Customer name:
+  ```
+
+### Step 3 - Generate An Ondat Licence
```

Comment:
The only bit I would mention is the fact that the portal can apply the licence automatically for you, so no need to generate ID or passed if that is the case. It might be worth mentioning it while leaving the content for those that need airgapped licences generated by the portal.

Reply:
@Arau Yes - I plan to explain this further in the licensing document > https://docs.ondat.io/docs/operations/licensing/ in a future PR instead - since we cover different methods on how to get and apply licences in that document.

Reply:
I have added a blurb on how automatically add a licence to the cluster through the SaaS portal.